rudder-web/src/main/scala/com/normation/rudder/appconfig/ConfigService.scala

```diff
@@ -329,6 +339,8 @@ class LDAPBasedConfigService(configFile: Config, repos: ConfigRepository, workfl
        rudder.policy.mode.name=${Enforce.name}
        rudder.policy.mode.overridable=true
        rudder.featureSwitch.directiveScriptEngine=enabled
+       rudder.compliance.unexpectedReportAllowsDuplicate=true
+       rudder.compliance.unexpectedReportUnboundedVarValues=true
```

Comment:
doesn't that mean that for upgrading user, the behavious will change ? Or will there be a migration script as well ?

Reply:
It does mean that. I believe it is the correct behavior as the previous behavior was in case that we are aware of considerated as a bug. I will need a special highligh in the changelog, but I really see no reason to not make that behavior the default one, even for migrating users.

Reply:
ok